x-pack/solutions/observability/packages/kbn-evals-suite-obs-ai-assistant/evals/esql/evaluate_esql_dataset.ts

```diff
@@ -78,8 +78,12 @@ export function createEvaluateEsqlDataset({
               .criteria([
                 ...(expected.expected
```

Comment:
this doesn't actually use the `criteria` defined in the object. I'm pretty sure that's my bad 😅 (I was going over some of the results)